### PR TITLE
pii handling for restore config

### PR DESCRIPTION
**Pull request by ryanfkeepers**
add pii concealer compliance for restore config structs.

---

#### Does this PR need a docs update or release note?

- [x] :no_entry: No

#### Type of change

- [x] :sunflower: Feature

### CODE REVIEW

File: src/pkg/control/restore.go [hunk 1]
```diff
@@ -90,3 +94,62 @@ func EnsureRestoreConfigDefaults(
...
+
+// Format produces a concealed representation of the config, even when
+// used within a PrintF, suitable for logging, storing in errors,
+// and other output.
+func (rc RestoreConfig) Format(fs fmt.State, _ rune) {
+	fmt.Fprint(fs, rc.concealed())
+}
+
+// String returns a string that contains all path config joined together.
+// Configs that need escaping are escaped.  The result is not concealed, and
```
Comment (ashmrtn) [inline src/pkg/control/restore.go:144]: this comment needs some revisions as it doesn't quite make sense

File: src/pkg/control/restore.go [hunk 2]
```diff
@@ -90,3 +94,62 @@ func EnsureRestoreConfigDefaults(
...
+// is not suitable for logging or structured errors.
+func (rc RestoreConfig) String() string {
+	return rc.PlainString()
+}
+
+// PlainString returns an unescaped, unmodified string of the joined elements.
```
Comment (ashmrtn) [inline src/pkg/control/restore.go:150]: This comment also could use some updates. RestoreConfig hasn't been using the terminology `elements` previously

**Review by ashmrtn — APPROVED** [review]

File: src/pkg/control/restore.go (hunk 1 above)
Comment (ryanfkeepers) [reply src/pkg/control/restore.go]: done

File: src/pkg/control/restore.go (hunk 2 above)
Comment (ryanfkeepers) [reply src/pkg/control/restore.go]: and done

**Pull request merged**